dbms/src/Storages/Page/V3/tests/gtest_page_directory.cpp

```diff
@@ -2057,6 +2057,36 @@ try
 }
```

Comment:
```suggestion
    {
        PageEntriesEdit edit;
        edit.del(buildV3Id(medium, 320));
        dir->apply(std::move(edit));
        ASSERT_EQ(dir->getMaxId(medium), 300);
    }

    {
        PageEntriesEdit edit;
        edit.del(buildV3Id(medium, 300));
        dir->apply(std::move(edit));
        ASSERT_EQ(dir->getMaxId(medium), 0);
    }
}
```

Reply:
This is to test delete with more than one `ns_id` in PageDirectory.

Reply:
done